requirements.txt

```diff
@@ -1,4 +1,4 @@
-git+https://github.com/cityofaustin/mds-provider-client.git@master#egg=mds_provider_client0.1.3
+git+https://github.com/cityofaustin/mds-provider-client.git@master#egg=mds_provider_client0.1.4
```

Comment:
what does 0.1.4 reference? Not seeing versions [here](https://github.com/cityofaustin/mds-provider-client/releases) or in the San Diego version. Seems like this `egg=` thing is specific to python requirement.txt files, but I still don't understand what its pointing at.

Reply:
The versioning is happening with the package definition in [`setup.py`](https://github.com/cityofaustin/mds-provider-client/blob/master/setup.py) for our mds-provider-client package. The version references the client itself, not an MDS version.

As far as I can tell, package versioning on github is not really useful. You can't `pip install` from an earlier version of the egg on the master branch. Publishing to package distribution (e.g. [pypi](http://pypi.org)) is different story. [Here's a tutorial](https://packaging.python.org/tutorials/packaging-projects/).

Good point that I should also add releases to the github release tracker.

Reply:
...to explain the `egg=`: so long as a repo as a `setup.py` and compliant directory structure, you can point `pip install` to a repo, and it will build and install the package egg from the contents of the repo. So you're seeing that magic path with `git+https://github.com/cityofaustin/mds-provider-client.git@master#egg=mds_provider_client0.1.4` in the requirements file.